backend/src/nodes/nodes/image_utility/convolve.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from . import category as ImageUtilityCategory
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput, TextInput
+from ...properties.outputs import ImageOutput
+from ...properties import expression
+
+
+@NodeFactory.register("chainner:image:image_convolve")
+class ImageConvolveNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Convolves input image with input kernel (kernel values separated by commas)"
+        self.inputs = [
+            ImageInput("Grayscale Image", channels=1),
+            TextInput("Kernel String"),
+            NumberInput("Kernel Width/Height", minimum=0, default=3),
+            NumberInput("Padding", minimum=0, default=0),
+            NumberInput("Strides", minimum=0, default=1),
```

Comment:
The minimum has to be 1.

```suggestion
            NumberInput("Strides", minimum=1, default=1),
```